zeekrunner

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+export ZEEKPATH=$dir/share/zeek:$dir/share/zeek/policy:$dir/share/zeek/site
+export ZEEK_PLUGIN_PATH=$dir/lib/zeek/plugins
+
+# The packet filter and loaded scripts are disabled because they emit either
+# timeless logs or logs with timestamp set to execution time rather than time
+# of capture.
+exec "$dir/bin/zeek" \
+  -C -r - \
+  --exec "event zeek_init() { Log::disable_stream(PacketFilter::LOG); Log::disable_stream(LoadedScripts::LOG); Log::disable_stream(Telemetry::LOG); }" \
```

Comment:
I spent some time looking at the additional logs that are produced by Zeek v6.0.2 relative to the ones that had been coming from our old Zeek v3.2.1 artifacts. Here I've added the [telemetry](https://docs.zeek.org/en/master/frameworks/telemetry.html) ones to our exclusion list since they create a lot of noise (e.g., 800+ additional log lines when processing a pcap that contains only a single connection) and are intended to give insight to operational deployments (i.e., not our use case.)